src/app/home/gets-started.js

```diff
@@ -0,0 +1,7 @@
+import GetStarted from "@/components/GetStarted";
+
+const Home = () => {
```

Comment:
The folder and files shouldn't be created. Please remove. Next meeting will explain why with next.js routing system.